doc/source/deployment/requirements.rst

```diff
@@ -51,13 +51,17 @@ Minimum Node Specification
   If the work node is used as Compute node, sizing shall be determined by
   the target workloads on the compute node.
 
-:term:`SES` AIO node (Experimental only)
-++++++++++++++++++++++++++++++++++++++++
+:term:`SES` All-In-One (AIO) node (Experimental only)
+++++++++++++++++++++++++++++++++++++++++++++++++++++
```

Comment:
add a + sign under close parenthesis

Reply:
Thanks for noticing it. Just addressed it.